tests/unit/test_estimator.py

```diff
@@ -990,6 +990,17 @@ def test_generic_deploy_accelerator_type(sagemaker_session):
     assert args[1][0]['InstanceType'] == INSTANCE_TYPE
 
 
+def test_deploy_with_update_endpoint(sagemaker_session):
+    e = Estimator(IMAGE_NAME, ROLE, INSTANCE_COUNT, INSTANCE_TYPE, output_path=OUTPUT_PATH,
```

Comment:
Avoid one letter variables.

Reply:
Updated. Do we need to update all other 1-letter variable names in this test? Many of the tests here use 'e' for estimator.